kzz7yh-f31379/cod-ve09v2_kzz7yh-f31379.xml

```diff
@@ -149,8 +150,9 @@
       adiuto<lb ed="#V" n="71" break="no"/>rium. Et dixi puro homini: quia nullus angelus fuit 
       depu<lb ed="#V" n="72" break="no"/>tatus ad custodiendum christum. Sed missi fuerunt 
       ange<lb ed="#V" n="73" break="no"/>li ad exhibendum sibi tanquam domino suo obsequium. 
+      <!--TODO: Add paragraph break-->
       <lb ed="#V" n="74"/>Unde primum argumentum concedendum est. 
-      <lb ed="#V" n="75"/>bene enim concedo quod christus 
+      <lb ed="#V" n="75"/>bene enim concedo quod christus <!--Abbreviation--> 
```

Comment:
christus is correct for xps (this abbreviations corresponds to the greek spelling of christ: xpistos
```suggestion
      <lb ed="#V" n="75"/>bene enim concedo quod christus
```